.github/workflows/benchmark.yaml

```diff
@@ -0,0 +1,59 @@
+name: Benchmark
+
+on:
+  push:
+    branches:
+      - benchmark
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Set up Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: 1.65.0
+          override: true
+
+      - name: Install Valgrind && Docker Compose
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y valgrind=1:3.18.1-1ubuntu2 &&
+          sudo apt-get install -y docker-compose
```

Comment:
this is not useful since no stratum server for sv1 (foundation server) is used in the benches

Reply:
Yes, you are right. I should I remove that part.